src/model/input.ts

```diff
@@ -8,129 +8,195 @@ const core = require('@actions/core');
  * Note that input is always passed as a string, even booleans.
  */
 class Input {
+  public static githubEnabled = true;
+  public static cliOptions;
+  static get cloudRunnerTests(): boolean {
+    return Input.getInput(`CloudRunnerTests`) || false;
+  }
+  private static getInput(query) {
+    return Input.githubEnabled
+      ? core.getInput(query)
+      : Input.cliOptions !== undefined && Input.cliOptions[query] !== undefined
+      ? Input.cliOptions[query]
+      : process.env[query] !== undefined
+      ? process.env[query]
+      : false;
+  }
+  static get region(): string {
+    return Input.getInput('region') || 'eu-west-2';
+  }
+  static get githubRepo(): string {
+    return Input.getInput('GITHUB_REPOSITORY') || 'game-ci/unity-builder';
+    // TODO system get repo remote?
+  }
+  static get branch() {
+    if (Input.getInput(`REMOTE_BUILDER_CACHE`)) {
+      return Input.getInput(`REMOTE_BUILDER_CACHE`);
+    } else if (Input.getInput(`GITHUB_REF`)) {
+      return Input.getInput(`GITHUB_REF`)
+        .split('/')
+        .filter((x) => {
+          x = x[0].toUpperCase() + x.slice(1);
+          return x;
+        })
+        .join('');
```

Comment:
I think you can simplify to ``.map(x => `${x[0].toUpperCase()}${x.slice(1)}`)`` here.

Just curious if you used `filter` over `map` for a reason here?

Reply:
Will replace with map